theme/src/styles/content-elements/big-icon-text-button/_index.scss

```diff
@@ -21,64 +21,93 @@
 // @use '../../bootstrap/mixins' as mixin;
 
 .big-icon-text-btn__wrp {
-    position: relative;
-    overflow: auto;
-    padding: 0 15px;
+  position: relative;
+  overflow: auto;
+  padding: 0 15px;
   margin: 0 -15px;
 }
 
 .big-icon-text-btn {
-    text-align: center;
-    padding: 50px 0 40px;
+  text-align: center;
+  padding: 50px 0 40px;
 }
 
 .big-icon-text-btn__link {
     // displays: flex;
-    position: relative;
-    z-index: 1;
-
-    &:after {
-        // @include icons();
+  position: relative;
+  z-index: 1;
 
-        position: relative;
-        display: inline-block;
-        font-size: 20px;
-        margin-left: 8px;
-        padding: 0 3px;
-        top: 1px;
-    }
+  &:after {
+    position: relative;
+    display: inline-block;
+    font-size: 20px;
+    margin-left: 8px;
+    padding: 0 3px;
+    top: 1px;
+  }
+
+  &.btn-primary {
```

Comment:
Better to use parent selector only with pseudo-classes, all other cases make it hard to read and maintain